File: SortingJava/BubbleSort.java
/*Bubble Sorting <Algorithm> */

class BubbleSort {
 
    static void bubbleSort(int arr[],int size)
    {
        int i,j;

        for(i = 0; i < size-1; i++)
        
            int flag = 0; 
        {
            for(j = 0; j < size-1-i; j++) 
            {
                if(arr[j] > arr[j+1])
                {
                    int t = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = t;
                    flag = 1 ; 
                }
            }
            
            if ( flag == 0 ) 
            
                break; 
        }
    }
    
    public static void main(String[] args) {
        
        int[] arr = new int[] {50, 45,37,69,25,13,42,78, 5,35,89,76, 20,27,18,56, 10,84,92};
        int n = arr.length;
        
        System.out.println("Array in sorted order : ");
        bubbleSort(arr,n);
        
        for(int i = 0; i < arr.length; i++)
        
            System.out.print(arr[i] + " ");
            
    }
    
}

File: SortingPyAlgo/BubbleSort.py
#Bubble >Sorting Alorithm ...

def bubbleSort(arr, size) :

	for i in range( size-1 ):

		flg = 0
		
		for j in range( size-1-i ) :
			
			if ( arr[j] > arr[j+1] ) :
			
				arr[j], arr[j+1] = arr[j+1], arr[j]

				flg = 1

		if ( flg == 0):

			break


if __name__=='__main__':


	arr = [ 45,67,86,36,13,78,94,56,34,17,81,32,48,52,24,15,79,85,35,19,27,84]
	n   = len(arr) 
	print("Len of Arr:\t",n)
	print(arr)
	bubbleSort(arr,n)
	print("\nSorted Array : \n")
	print(arr)
